java-components/sidecar/src/main/java/com/redhat/hacbs/sidecar/resources/HashUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.hacbs.sidecar.resources;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public final class HashUtil {
+
+    private HashUtil() {
+    }
+
+    public static String sha1(String value) {
+        return sha1(value.getBytes(StandardCharsets.UTF_8));
+    }
+
+    public static String sha1(byte[] value) {
+        try {
+            MessageDigest md = MessageDigest.getInstance("SHA-1");
+            byte[] digest = md.digest(value);
+            StringBuilder sb = new StringBuilder(40);
+            for (int i = 0; i < digest.length; ++i) {
+                sb.append(Integer.toHexString((digest[i] & 0xFF) | 0x100).substring(1, 3));
+            }
+            return sb.toString();
```

Comment:
If we're using Java 17, it has a built in method for this now https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/HexFormat.html#formatHex(byte%5B%5D)

Reply:
Ok, this was existing code that just moved to a Util class for reuse.